### DISCUSSION TITLE
21.08.04.수 회의록

### DISCUSSION THREAD

**Discussion by damilog** [Team-IT-DA/Frontend#36] [회의록]
- [x] product 페이지에서 GET /api/products 할 때  "description": "제주 바다 바람을 품은 친환경 흙당근" 데이터도 가져오는 것에 대한 논의(백엔드분들께 요청해야함), 아래 처럼 상품 상세 내역도 함께 보여주는게  어떠한지? (데이지) -> 백엔드 분들께 요청
<img width="298" alt="스크린샷 2021-08-04 오전 1 06 30" src="https://user-images.githubusercontent.com/56783350/128048930-7cc6d900-6728-4ed8-8706-7d1c85bb6d57.png">

- [x] product 페이지에서 product card 내에서 보여지는 이미지의 크기를 강제로 정할 건지 아니면 업로드 할 때 정해진 규격에 맞게 업로드 하게 할건지? (데이지) -> 고정값으로 가는걸로 (product card는 우선 데이지가 사용하고 추후에 cart에 쓰이는 product card는 추가 예정)
 <img width="763" alt="스크린샷 2021-08-04 오전 1 02 58" src="https://user-images.githubusercontent.com/56783350/128048395-452a5139-f6a2-42f5-b7e3-0b0cc7e21768.png">

- [x]  product card 수정 사항 공유 (스타일 수정 필요한 부분, 장바구니 버튼 추가 괜찮은지, 데이지가 추가한 부분)

- [x] product card 내 장바구니 버튼 클릭 후 보여지는 모달창 추가에 대한 논의 
<img width="478" alt="스크린샷 2021-08-04 오전 1 10 41" src="https://user-images.githubusercontent.com/56783350/128049543-73b981a1-a1b4-4fed-a43e-88ab4dfb547f.png">
 
- [x] 상품 디테일 페이지 컴포넌트 설계 내용 공유 (제니)
- [x] 각자 이번주 목표치 공유, 다음 미팅 일정 정하기

**Comment by ha3158987** [comment]
<💡 공지>
PR에 `Resolved #이슈번호`로 입력하면 해당 이슈 자동 closed 되도록 default 브랜치를 develop으로 바꿔놓았습니다!!

**Comment by damilog** [comment]
<제니>
- ICommonInfo : id, imageUrl 등등 자주 쓰이는 interface 만들 예정
- product detail 컴포넌트 설계 내용 피그마에 업데이트

**Comment by damilog** [comment]
일요일 까지 목표치 
<제니> 
- 리뷰창 view 
- 리뷰창 포토 후기 일반 후기 view 
- tab 전환시 scroll 제일 하단으로내리기 (시간 되면) 

<데이지>
- lazy Loading 구현
- prodcut page view 완성 (card 수정)
- 장바구니 모달창 view 구현(시간 되면)
